### PR TITLE
DESKTOP VERSION OF PORTFOLIO-WEBSITE

### PR DESCRIPTION
**Pull request by citec-47**
###SET UP THE PROJECT 🌳
-Required a desktop of at least 4RAM, a web-browser e.g firefox, google chrom, a code editor e.g VS CODE, SUMBLIME, and the programming languages used where HTML and CSS

##General requirements🧾
-Make sure that there are [no linter errors] 
-Make sure that you used correct

 #HTML/CSS requirements
-Used best practices for HTML & CSS
#Project requirements🌳

-Desktop✔
-Use media queries in your CSS file to make your desktop version covers the dimension (Desktop: >=768px).
-In order to lay out the elements on the page you should use Grid and Flexbox.
-You don't need to implement any functionality that requires JavaScript.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @citec-47 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks a descriptive README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by citec-47** [comment]
#README FILE
-adjustment of my readme file

**Review by amejid — COMMENTED** [review]
Hi @citec-47, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, where the PR description is directly copied from the project requirements. Kindly put a summary of the introduced changes inside your PR description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by citec-47** [comment]
#adjustment of the PR form

File: style.css [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+  background: #6070ff;
+}
+
+.back_image {
+  background-image: url('Images/background_img.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  max-width: 100%;
+  height: auto;
+}
```
Comment (levy002) [inline style.css:72]: - [ ] Kindly make sure to align the content of this container correctly in order to meet the Figma design as much as possible. You can use CSS flexbox to achieve that.
- [ ] Kindly make sure to fix this headline section on the mobile version.

   - ### YOUR DESIGN
   
    image

   image

   - ### FIGMA DESIGN
   
   image

   image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+  }
+
+  #textpanel {
+    width: 50%;
+    border-radius: 0;
+  }
+
+  .submit {
+    border-radius: 0;
+  }
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure to add the `border-radius` on the `submit` button in order to meet the Figma design as much as possible.

  - ### YOUR DESIGN
  
     image

  - ### FIGMA DESIGN
  
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @citec-47   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to fix the styling on the `Get my resume` button on the mobile version because now it now looking like the one on the Figma design.

   - ### YOUR DESIGN
   
    image

   - ### FIGMA DESIGN
   
    image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by citec-47** [comment]
#Changes
[]styling on the Get my resume button
[]fixing of the header section on of the mobile version
[]

**Review by frank1738 — APPROVED** [review]
Hi @citec-47  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Documentation 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**